lab-anthony/test/greet-test.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const getGreet = require(__dirname + '/../lib/greet');
+const assert = require('assert');
+
+
```

Comment:
However this first test will fail if any program runs mocha for you because process.argv's lenght will change